package.json

```diff
@@ -1,15 +1,18 @@
 {
-  "name": "bodylabs-javascript-style",
-  "version": "6.0.0",
+  "name": "eslint-plugin-bodylabs",
```

Comment:
Unfortunately yes, it's a requirement of eslint that plugins be named this way. A bit gross, but it's the way that it is. I'll publish a new package under the new name so we can keep pulling it in from npm, and rename the repo to minimize confusion. (I'm keeping the semver sequence same so we can use the same repo.)